payment/src/main/java/com/kyriba/conference/payment/api/dto/PaymentMethodResponse.java

```diff
@@ -0,0 +1,9 @@
+package com.kyriba.conference.payment.api.dto;
+
+import com.kyriba.conference.payment.domain.PaymentMethodType;
+import lombok.Value;
+
+@Value
+public class PaymentMethodResponse {
```

Comment:
could be placed in controller

Reply:
I prefer to keep all dto in a single place.

Reply:
I will consider for the future